packages/video_player/test/video_player_test.dart

```diff
@@ -41,6 +41,9 @@ class FakeController extends ValueNotifier<VideoPlayerValue>
   Future<void> play() async {}
   @override
   Future<void> setLooping(bool looping) async {}
+
+  @override
+  VideoFormat get formatHint => null;
```

Comment:
I just realized that we don't have any tests on the `initialize`
Do you mind to add one here? Just to test the formatHint is pass to the method channel correctly.

Reply:
I'd like to put that in another CL as we're trying to integrate with this at the moment.

Reply:
Sounds good! Thanks.